src/api/localStorage.js

```diff
@@ -0,0 +1,111 @@
+import Enumerable from '../../node_modules/linq';
+import faker from '../../node_modules/faker';
+export class LocalStorage {
+  constructor(_id) {
+    this._id = _id;
```

Comment:
Don't think having a class property for `_id` is needed, and then consequently the constructor itself. Have the functions work with passed in ids rather than `this._id`

Reply:
Or, with that said, consider using the `constructor` to create/retrieve and parse the user information so that you can skip that step within your functions.